kpatch-build/create-diff-object.c

```diff
@@ -2109,6 +2148,9 @@ void kpatch_create_hooks_objname_rela(struct kpatch_elf *kelf, char *objname)
 	}
 }
 
+#ifdef __powerpc__
+void kpatch_create_mcount_sections(struct kpatch_elf *kelf) { }
```

Comment:
- load_module
  - ftrace_module_init
     - ftrace_process_locs
        - ftrace_update_code
           - ftrace_code_disable
              - ftrace_make_nop
                this function expects the opcode, to be valid bl (branch op). Converting it to nop manually in the patched module will result it crash. As it perform the check for valid branch using (op & 0xfc000003) == 0x48000001